src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,6 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
+        "symfony/expression-language": "",
```

Comment:
@stof As you surely guessed, I took the existing ones as example, but I totally agree so I'll add one.